libs/template/application/lambda/src/handler.lambda.ts

```diff
@@ -44,7 +45,9 @@ export const storefrontHandler = async (
         ...event.headers,
       },
       body,
-      ttl: process.env.ORYX_TTL ? Number(process.env.ORYX_TTL) : ttl,
+      ...(!hasQueryParams
+        ? { ttl: process.env.ORYX_TTL ? Number(process.env.ORYX_TTL) : ttl }
+        : {}),
```

Comment:
Let's not play with TTL for this. I'd rather turn params into the path as we want to have it anyways for better UX and SEO.